src/EnergyPlus/PluginManager.cc

```diff
@@ -1388,6 +1399,8 @@ namespace PluginManagement {
 #if LINK_WITH_PYTHON == 1
     void PluginManager::addToPythonPath(const std::string &path, bool userDefinedPath)
     {
+        if (path.empty()) return;
```

Comment:
And here we don't try to add a blank path to the Python search path.  :heavy_check_mark: